pkg/controllers/node/node_controller.go

```diff
@@ -193,3 +230,207 @@ func kick(c chan<- interface{}) {
 		// pass
 	}
 }
+
+// generateHostendpointName returns the auto hostendpoint's name.
```

Comment:
There's a lot of auto-hep specific stuff here. For the other things this controller does, I think we've put them in separate files.

Maybe we should have a `hostendpoints.go` file where we can put all the auto-HEP logic?

Reply:
Done! f919ddc40c46b8b3aaad02cfb0faa81f1bdb84f6